lib/tasks/peer_review.rake

```diff
@@ -0,0 +1,36 @@
+# NOTE: The assignment cannot be generated in the assignments rake task since
+# it causes errors to occur. I guess this is due to it not being populated
+# properly and something errors out. Therefore, all the peer review stuff is
+# done here, including but not limited to creating the PR assignment.
+
+namespace :db do
+  desc 'Create peer reviews and related data'
+  task :peer_reviews => :environment do
+    puts 'Creating A1 Peer Review and updating A1 to have the peer review'
+    a1 = Assignment.find_by(short_identifier: 'A1')
+    a1.has_peer_review = true  # Creates 'a1pr' via callback.
+    a1.save
+
+    a1pr = Assignment.find_by(short_identifier: a1.short_identifier + '_pr')
```

Comment:
Better to do `a1pr = a1.peer_review_assignment`, or whatever the association is called.